src/systems/ec/implementations/react/packages/radio/src/RadioButton.jsx

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classnames from 'classnames';
+
+const RadioButton = ({
+  id,
+  name,
+  value,
+  disabled,
+  className,
+  helperText,
+  label,
+  ...props
+}) => {
+  const classNames = classnames(className, 'ecl-radio', {
+    'ecl-radio--disabled': disabled,
+  });
+
+  return (
+    <div {...props} className={classNames}>
+      <input
+        id={id}
+        name={name}
+        className="ecl-radio__input"
+        type="radio"
+        value={value}
+        disabled={disabled}
+      />
+      <label htmlFor={id} className="ecl-radio__label">
+        <span className="ecl-radio__box" />
+        {label}
+      </label>
+      {helperText && (
+        <div className="ecl-radio__help ecl-help-block">{helperText}</div>
```

Comment:
Why not a p?
Why not using aria-describedby?
https://developer.paciellogroup.com/blog/2014/12/using-aria-describedby-to-provide-helpful-form-hints/
general guidelines for "instructions"
https://www.w3.org/WAI/tutorials/forms/instructions/